SecondSight/config.py

```diff
@@ -2,6 +2,22 @@
 
 import sys
 import json
+import os
+
+
+def loadConfig():
```

Comment:
We probably shoudln't use this function. The class is already a singleton, just add the ability to know if there is a config file in the class itself, which helps make loading the config data exactly the same no matter where it happens